src/Elastic/Serializer.elm

```diff
@@ -1,50 +1,58 @@
-module Elastic.Serializer exposing (run)
+module Elastic.Serializer exposing (Config(..), run)
 
 import Elastic.Expression exposing (Expr(..))
 
 
-group : Expr -> String
-group expr =
-    "(" ++ run expr ++ ")"
+type Config
+    = Config Bool
```

Comment:
Why not a record? what if we want to add more options later on? I'd suggest we keep the opaque type though.